styleguide/components/61-layouts/general/info-layout/info-layout.twig

```diff
@@ -31,8 +36,64 @@
       "text": "Nam lorem ex, posuere non lectus tempus, ornare scelerisque mauris. Integer vitae tortor orci. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Maecenas enim lectus, malesuada in hendrerit in, vulputate ut enim. Integer pretium elit purus, ac pulvinar augue scelerisque quis. In velit elit, luctus vitae nunc quis, hendrerit congue felis. Nulla tincidunt tincidunt cursus. Mauris pretium eu ex id aliquet. Aenean massa metus, posuere ac dui sit amet, volutpat interdum libero."
     } %}
 
+    <div class="video wider">
+      <iframe width="100%" height="500" src="https://www.youtube.com/embed/S0vLmTWUmok" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+    </div>
+
     {% include '@feedback-form' with {
       'modifier': 'wider'
     }%}
+
+  </div>
+  <div class="extra-info-box">
```

Comment:
Suggest using 'aside' tag. 

"The HTML <aside> element represents a portion of a document whose content is only indirectly related to the document's main content."

Reply:
Agreed.